pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>se.idsec.signservice.integration</groupId>
   <artifactId>signservice-integration-api</artifactId>
-  <version>2.0.0</version>
+  <version>2.1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
```

Comment:
Dependencies to Jackson and jakarta are too old and will collide with the ones from the SignService commons BOM.